addons/zylann.hterrain/tools/importer/importer_dialog.gd

```diff
@@ -276,16 +276,18 @@ static func _load_image_size(path: String, logger) -> HT_ImageSizeResult:
 			result.error_code = err
 			return result
 
-		# Assume the raw data is square in 16-bit format,
+		# Assume the raw data is square in 16-bit or 24-bit format,
```

Comment:
We should not assume bit number, we should know that we are importing 16 or 24 bits.

Reply:
Done: upon file choosing, try to auto-detect bit depth and set it as the default value. But the user can change it via OptionButton

![image](https://github.com/Zylann/godot_heightmap_plugin/assets/6639237/00f927e5-2da7-46dc-889e-9fcce45fd38a)